base/essentials.jl

```diff
@@ -337,3 +337,16 @@ function vector_any(xs::ANY...)
 end
 
 isempty(itr) = done(itr, start(itr))
+
+"""
+    invokelatest(f, args...)
+
+Calls `f(args...)`, but guarantees that the most recent method of `f`
+will be executed.   This is useful in specialized circumstances,
+especially `cfunction` callbacks that may extract a Julia `Function`
```

Comment:
I was thinking of something like https://github.com/JuliaGizmos/Reactive.jl/issues/131, where it's not calling eval, but it wants to be able to react when other Tasks call eval.